plugins/normalize.js

```diff
@@ -0,0 +1,9 @@
+const plugin = require("tailwindcss/plugin");
+
+module.exports = plugin(function ({ addBase }) {
+  addBase({
```

Comment:
I like this but I think it should be in the tailwind.config.js file.... similar to https://play.tailwindcss.com/uF9rN9jewc?file=config

Reply:
Done - plugin removed

Base font has been defined in style.css